packages/react/src/ReactFetch.js

```diff
@@ -74,7 +74,9 @@ if (enableCache && enableFetchInstrumentation) {
         url = resource;
       } else {
         // Normalize the request.
-        const request = new Request(resource, options);
+        // if resource is not a string (its an instance of Request)
+        // we clone the request as to not disturb the body (in the event it's a ReadableStream)
+        const request = typeof resource === 'string' ? new Request(resource, options) : resource.clone();
```

Comment:
It seems better to just let the request pass through in this case instead of cloning it.

The normalization is more about turning the string case into a request so we can get the right keys.